### PR TITLE
Change scenes buttons ripple effect to white #208

### PR DESCRIPTION
**Pull request by chena-1**

### CODE REVIEW

File: lib/presentation/scenes/widgets/scene_widget.dart [hunk 1]
```diff
@@ -30,25 +30,27 @@ class SceneWidget extends StatelessWidget {
...
+                shape: RoundedRectangleBorder(
+                  borderRadius: BorderRadius.circular(30),
+                  side: BorderSide(
+                      color: (Theme.of(context).textTheme.bodyText1!.color)!,
+                      width: 0.6),
+                ),
               ),
               onPressed: () {
                 HapticFeedback.lightImpact();
+
```
Comment (guyluz11) [inline lib/presentation/scenes/widgets/scene_widget.dart:46]: Please remove extra space

**Review by guyluz11 — COMMENTED** [review]
Check review commit description and update pull request.

**Comment by guyluz11** [comment]
Please check the commit review description

**Comment by guyluz11** [comment]
@chena-1

**Comment by chena-1** [comment]
line removed

**Pull request merged**